HeterogeneousCore/SonicTriton/src/TritonClient.cc

```diff
@@ -0,0 +1,318 @@
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "HeterogeneousCore/SonicTriton/interface/TritonClient.h"
+
+#include "request_grpc.h"
+
+#include <string>
+#include <cmath>
+#include <chrono>
+#include <exception>
+#include <sstream>
+#include <cstring>
+
+namespace ni = nvidia::inferenceserver;
+namespace nic = ni::client;

```

Comment:
Does
```suggestion
namespace {
  namespace ni = nvidia::inferenceserver;
  namespace nic = ni::client;
}
```
work? If I understood correctly, then the names `ni` and `nic` are not exported outside of this file.

Reply:
I did this in the header, but I thought there wasn't a problem with namespace aliases in a .cc file.

Reply:
Maybe it isn't a problem after all, the actual symbols use the real namespace name, and whatever shorthands are (should?) not be visible outside of the translation unit.